server/routes/dischargeAddress.js

```diff
@@ -16,16 +16,16 @@ module.exports = function({logger, dischargeAddressService, licenceService, auth
         logger.debug('GET /dischargeAddress');
 
         const nomisId = req.params.nomisId;
-        const licence = await licenceService.getLicence(nomisId);
+        const rawLicence = await licenceService.getLicence(nomisId);
```

Comment:
Youre not handling the error from the promise `licenceService.getLicence(nomisId);`
Simplest solution is to wrap this in a try catch